src/controllers/health.js

```diff
@@ -17,5 +18,11 @@ const model = require('../models/health')
  * @returns {void}
  */
 exports.getResource = function * health () {
-  this.body = yield model.getDbHealth()
+  const now = Date.now()
+  // call model.getDbHealth(...) max. once per minute
+  if (!lastQueried || lastQueried > (now - 60000)) {
```

Comment:
Constants like the `60000` should be declared at the top of the file as `const`'s